chirp/drivers/baofeng_uv17.py

```diff
@@ -158,6 +158,10 @@ class UV17(baofeng_uv17Pro.UV17Pro):
 
     MODES = ["FM", "NFM"]
     BLOCK_ORDER = [16, 17, 18, 19,  24, 25, 26, 4, 6]
+
+    # Add extra read blocks (e.g. calibration block 2) to the end here:
+    BLOCK_O_READ = BLOCK_ORDER
```

Comment:
This does not make a copy. If I modify `BLOCK_ORDER` I'm also modifying `BLOCK_O_READ`. You need to do something like this:
```
BLOCK_O_READ = list(BLOCK_ORDER)
```